tree/ntuple/v7/test/CustomStruct.hxx

```diff
@@ -17,6 +18,14 @@ struct CustomStruct {
    std::vector<float> v1;
    std::vector<std::vector<float>> v2;
    std::string s;
+
+   bool operator <(const CustomStruct& c) const {
+      return a < c.a;
+   }
+
+   bool operator==(const CustomStruct& c) const {
+      return a == c.a && v1 == c.v1 && v2 == c.v2 && s == c.s;
```

Comment:
It seems somewhat inconsistent to consider only the `a` member for `operator<`, but all members for equality.